### PR TITLE
REDESIGN: [7628] BO landmark page structure

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

File: meinberlin/apps/projects/templates/meinberlin_projects/project_bplan_detail.html [hunk 1]
```diff
@@ -17,15 +17,16 @@
...
-                    {% html_date project.modules.first.future_phases.first.start_date 'DATE_FORMAT' as start_date %}
-                    {% blocktranslate with date=start_date %}It starts on {{ date }}.{% endblocktrans%}
-                </p>
-            </div>
-        </div>
+
+        <h1 class="u-first-heading">{% translate 'Participation is not possible at the moment.' %}</h1>
+        <p>
+            {% html_date project.modules.first.future_phases.first.start_date 'DATE_FORMAT' as start_date %}
+            {% blocktranslate with date=start_date %}It starts on {{ date }}.{% endblocktrans%}
```
Comment (m4ra) [inline meinberlin/apps/projects/templates/meinberlin_projects/project_bplan_detail.html:28]: i know is from old code, but here too an extra space b4 closing tag... :)

**Review by m4ra — COMMENTED** [review]
a lil nitpicking

File: meinberlin/apps/bplan/templates/meinberlin_bplan/statement_sent.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {% block title %}{% translate 'Statement sent' %} &mdash; {{ block.super }}{% endblock %}
 
 {% block content %}
-<div class="container">
+<div id="layout-grid__area--maincontent">
```
[the remark this answers is not in the document]
Comment (philli-m) [reply meinberlin/apps/bplan/templates/meinberlin_bplan/statement_sent.html:7]: @hom3mad3 no unfortunately not, I don't like the use of id's for styling but dues to their nesting it's unavoidable :(

File: meinberlin/apps/ideas/templates/meinberlin_ideas/idea_moderate_form.html [hunk 3]
```diff
@@ -2,27 +2,25 @@
...
 
 {% block title %}{{ object.name }} &mdash; {% translate 'Give feedback' %}{% endblock %}
-
 {% block breadcrumbs %}
     <div id="layout-grid__area--contentheader">
         <div id="content-header">
             <nav class="breadcrumb" aria-label="{% translate 'You are here:' %}">
                 <ol>
-                    <li><a href="{% url 'meinberlin_plans:plan-list' %}">{% translate 'Project Overview' %}</a></li>
+                    <li><a href="{% url 'meinberlin_plans:plan-list' %}">{% trans 'Project Overview' %}</a></li>
```
[the remark this answers is not in the document]
Comment (philli-m) [reply meinberlin/apps/ideas/templates/meinberlin_ideas/idea_moderate_form.html:10]: err sorry! I will start running a search before i push anything

**Comment by philli-m** [comment]
@hom3mad3 nice catch! that was actually everywhere, I copied it from BO styleguide and apparently they use german quotes sometimes?! will look into a linter for our templates so doesn't happen again

**Comment by philli-m** [comment]
got approval so will merge

**Pull request merged**